scl-rubygem-GEM.spec.template

```diff
@@ -0,0 +1,118 @@
+
+Name:    %{?scl_prefix}rubygem-%{gem_name}
+Version: <%= spec.version %>
+Release: 1%{?dist}
+Summary: <%= spec.summary.gsub(/\.$/, "") %>
+Group:   Development/Languages
+License: <%= spec.licenses.join(" and ") %>
+<% if spec.homepage -%>
+URL:     <%= spec.homepage %>
+<% end -%>
+Source0: <%= download_path %>%{gem_name}-%{version}.gem
+
+BuildRequires: %{?scl_prefix_ruby}ruby(release)
+<% for req in spec.required_rubygems_version -%>
+BuildRequires: %{?scl_prefix_ruby}<%= requirement 'rubygems-devel', req %>
+<% end -%>
+<% for req in spec.required_ruby_version -%>
+BuildRequires: %{?scl_prefix_ruby}<%= requirement "ruby#{'-devel' unless spec.extensions.empty?}", req %>
+<% end -%>
+<% for d in spec.development_dependencies -%>
+<% unless ["rdoc", "rake", "bundler"].include? d.name -%>
+<% for req in d.requirement -%>
+BuildRequires: %{?scl_prefix_ruby}<%= requirement "rubygem(#{d.name})", req %>
+<% end -%>
+<% end -%>
+<% end -%>
+<% if spec.extensions.empty? -%>
+BuildArch: noarch
+<% end -%>
+Provides: %{?scl_prefix}rubygem(%{gem_name}) = %{version}
+
+%{?scl:Requires: %{?scl_prefix}runtime}
+
+%description
+<%= spec.description %>
+
+<% if doc_subpackage -%>
+%package doc
+Summary:   Documentation for %{name}
+Group:     Documentation
+Requires:  %{?scl_prefix}%{pkg_name} = %{version}-%{release}
+BuildArch: noarch
+
+%description doc
+Documentation for %{name}.
+<% end # if doc_subpackage -%>
+
+%prep
+%setup -n %{pkg_name}-%{version} -q -c -T
+%{?scl:scl enable %{scl} - << \EOF}
+%gem_install -n %{SOURCE0}
+%{?scl:EOF}
+
+%build
+
+%install
+mkdir -p %{buildroot}%{gem_dir}
+cp -a .%{gem_dir}/* \
+        %{buildroot}%{gem_dir}/
+
+<% unless spec.extensions.empty? -%>
+mkdir -p %{buildroot}%{gem_extdir_mri}
+cp -a .%{gem_extdir_mri}/{gem.build_complete,*.so} %{buildroot}%{gem_extdir_mri}/
+
+<% for ext in spec.extensions -%>
+# Prevent dangling symlink in -debuginfo (rhbz#878863).
+rm -rf %{buildroot}%{gem_instdir}/<%= ext.split(File::SEPARATOR).first %>/
+<% end -%>
+<% end -%>
+
+<% unless spec.executables.nil? or spec.executables.empty? -%>
+mkdir -p %{buildroot}%{_bindir}
+cp -pa .%{_bindir}/* \
+        %{buildroot}%{_bindir}/
+<% end -%>
+
+<% unless spec.executables.empty? -%>
+find %{buildroot}%{gem_instdir}/bin -type f | xargs chmod a+x
+<% end -%>
+
+# Run the test suite
+%check
+pushd .%{gem_instdir}
+
+popd
+
+%files
+%dir %{gem_instdir}
+<% unless spec.executables.nil? or spec.executables.empty? -%>
+<% for f in spec.executables -%>
+%{_bindir}/<%= f %>
+<% end -%>
+<% end -%>
+<% unless spec.extensions.empty? -%>
+%{gem_extdir_mri}
+<% end -%>
+<%= main_file_entries(spec) %>
+<% unless doc_subpackage -%>
+%doc %{gem_docdir}
+<%= doc_file_entries(spec) -%>
+<% end -%>
+%exclude %{gem_cache}
+%{gem_spec}
+
+<% if doc_subpackage -%>
+%files doc
+%doc %{gem_docdir}
+<%= doc_file_entries(spec) %>
+<% end # if doc_subpackage -%>
+
+%changelog
+* <%= Time.now.strftime("%a %b %d %Y") %> <%= packager %> - <%= spec.version %>-1
```

Comment:
If we tag with tito, is this entry still needed?

Reply:
I'd like that `if` condition to be false. I think I have come around to `tito tag` being superfluous and would make this workflow easier. For example, adding a job that auto-builds any changes when merged into the Copr repository. Avoiding `tito tag` makes this much easier.